service/src/main/java/skills/controller/PublicConfigController.groovy

```diff
@@ -66,6 +66,9 @@ class PublicConfigController {
     @Value('${skills.config.ui.enablePageVisitReporting:#{false}}')
     Boolean enablePageVisitReporting
 
+    @Value('${spring.security.saml2.registrationId:saml2TestRegId}')
```

Comment:
Instead of defaulting the test value can you please specify this property in the test itself (`service/src/test/java/skills/intTests/saml2/SAML2IntegrationIT`)?